Telereso/Classes/Extensions/TButtonExtension.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  TButtonExtension.swift
+//  Telereso
+//
+//  Created by Ganesh TR on 04/05/22.
+//
+
+import SDWebImage
+
+extension UIButton {
+
+    public func setRemoteImageWith(key: String, for state: UIControl.State) {
```

Comment:
i feel `setRemoteImage` , is more coherent with both `UIButton.setImage` and `sd_setImage`, 

same for other extension functions